plugins/base/src/main/resources/dokka/styles/style.css

```diff
@@ -953,8 +953,8 @@ td.content {
     padding-bottom: 8px;
 }
 
-.table-row .with-platform-tabs .sourceset-dependent-content .brief,
-.table-row .with-platform-tabs .sourceset-dependent-content .inline-comment {
+.table-row .platform-hinted .sourceset-dependent-content .brief,
+.table-row .platform-hinted .sourceset-dependent-content .inline-comment {
```

Comment:
Because there's no tabs, it renders the pages the same way it would for single platform projects, and there's currently the same problem - no padding. See [this page](https://nomisrev.github.io/kotlinx-serialization-jsonpath/-kotlin-x%20-serialization%20-json-path/io.github.nomisrev/index.html#993988515%2FFunctions%2F788909594) for instance

This should solve it for both single and multi platform pages